test_scenarios/T3_Create_New_CT.md

```diff
@@ -0,0 +1,40 @@
+# Test Scenario
+
+## T1 Stworzenie nowego CT
+
+### Sprawdzane UC
+
+- [UC6](../scenarios/UC6_Create%20CT)
+
+### Dane początkowe
+
+![model](data/T3_in.svg)
+
+### Warunki początkowe 
+
+- Zalogowany __App User__ to _user01_.
+
+### Przypadki testowe
+
+#### 1. Stworzenie nowego CT
+
+1. __App User__ wybiera przycisk "Add Computation Task"
+2. System wyświetla listę aplikacji zawierającą przynajmniej "Test App 01"
+3. __App User__ kilka przycisk "Choose" przy applikation "Test App 01"
```

Comment:
Skąd to wziąłeś?

Reply:
Jest opisane w w podlinkowanym `![model](data/T3_in.svg)`. Jak wejdziesz na branchu w wyrenderowaną wersję, to będzie widać.

Reply:
Chodziło mi o przycisk 'Choose'.

Reply:
Chodziło mi o przycisk 'Choose'.

Reply:
Chodziło mi o przycisk 'Choose'.

Reply:
Chodziło mi o przycisk 'Choose'.

Reply:
Zmieniłem